src/components/pages/GalleryPage.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
this is new shared component, it should allow for a bit less redundancy in each of the gallery pages.